AutoMad/wordFormer.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Fri Nov  1 13:47:53 2024
+
+@author: dragon
+"""
+import pyinflect
+
+
+"""def inflectCorrect(word, possum):
+    e = inflect.engine()
+    if possum in ['NNS','NNPS']:
+        return e.plural(word)
+    if possum == 'VBG':
+        return e.present_participle(word)
+    if possum == 'VBD':
+        return e."""
+    
+def inflectCorrect(word, possum):
```

Comment:
This function could use a docstring it's not obvious what it does without going and reading pyinflect's documentation (even just linking to said documentation would be helpful)